ansible_collections/f5networks/f5_modules/plugins/modules/bigip_profile_fastl4.py

```diff
@@ -284,6 +284,10 @@
     description:
       - Enables or disables hardware SYN cookie support when PVA10 is present on the system.
     type: bool
+  syn_cookie_enable:
+    description:
+      - Specifies whether or not to use SYN Cookie.
+    type: bool
```

Comment:
missing version_added

Reply:
Added version

Reply:
this needs to be "1.11.0" as its new feature only bugs increment the last digit, which is why most likely you have sanity error, also i think your indentation is a bit off recheck it